source/agora/common/crypto/ECC.d

```diff
@@ -298,9 +298,13 @@ public struct Point
         this.data = data;
     }
 
-    /// Construct a point from its string representation or a `ubyte[]`
-    public this (T) (T param)
-        nothrow @nogc
+    public bool isValid () nothrow @nogc @trusted const
```

Comment:
Could you move this after all the constructors? All overloads (including constructors) should be grouped together. And please add some docs too! :)

Reply:
Done.